File: test/fixtures/langs/sample.cs
public class Calc
{
    //∷YAY⟨CS-1⟩
    //  unit: Add
    //  intent: add two integers
    //  in: (a:number, b:number)
    //  out: number
    //  ensures: out === a + b
    //∷YAY-END⟨CS-1⟩
    public int Add(int a, int b)
    {
        return a + b;
    }

    public int Undocumented(int x)
    {
        if (x > 0) { return x; }
        return -x;
    }
}

File: test/fixtures/langs/sample.rs
//∷YAY⟨RS-1⟩
//  unit: add
//  intent: add two integers
//  in: (a:number, b:number)
//  out: number
//  ensures: out === a + b
//∷YAY-END⟨RS-1⟩
pub fn add(a: i64, b: i64) -> i64 {
    a + b
}

fn undocumented(x: i64) -> i64 {
    if x > 0 { x } else { -x }
}
